pkg/apis/node/v1beta1/defaults.go

```diff
@@ -0,0 +1,51 @@
+/*
+Copyright 2019 The Kubernetes Authors.
+
+Licensed under the Apache License, Version 2.0 (the "License");
+you may not use this file except in compliance with the License.
+You may obtain a copy of the License at
+
+    http://www.apache.org/licenses/LICENSE-2.0
+
+Unless required by applicable law or agreed to in writing, software
+distributed under the License is distributed on an "AS IS" BASIS,
+WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+See the License for the specific language governing permissions and
+limitations under the License.
+*/
+
+package v1beta1
+
+import (
+	"k8s.io/api/core/v1"
+	"k8s.io/api/node/v1beta1"
+)
+
+func SetDefaults_Overhead(obj *v1beta1.Overhead) {
```

Comment:
Validation and Defaulting need to be registered. I can help you set it up next week, but first I think we should discuss whether requests and limits both make sense for overhead.